app/views/posts/index.html.erb

```diff
@@ -1,55 +1,35 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Weblog</title>
-    <meta name="viewport" content="width=device-width,initial-scale=1">
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-
-    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
-    <%= javascript_importmap_tags %>
-  </head>
-
-  <body>
     <div class="postCont">
       <div class="pic"></div>
        <div class='userCard'>
-        <h2>John Doe</h2>
-        <h3>Number of posts: X</h3>
+        <h2><%= @user.name %></h2>
+        <h3>Number of posts:  <%= @posts.length %> </h3>
        </div>
     </div>
 
+  <% @posts.each do |post| %>
   <div class='post'>
-  <h2>Post 1</h2>
+  <h2><%= post.title %></h2>
   <p>
-  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin sed volutpat lorem, in feugiat tellus. Nulla facilisi. 
+    <%= post.text %>
   </p>
-  <h3>Number of posts: X</h3>
-  </div>
+  <h3>
+    Comments: <%= post.comments.length %> 
+    Likes: <%= post.likes.length %> 
+  </h3>
 
+  <% post.comments.each do |com| %>  
   <div class='comment'>
-    <p>Tyler: Nice comment!</p>
-    <p>John: Cool way to put it</p>
-    <p>Fernando: LOL!</p>
-  </div>
-  
-  <div class='post'>
-  <h2>Post 2</h2>
-  <p>
-  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin sed volutpat lorem, in feugiat tellus. Nulla facilisi. 
-  </p>
-  <h3>Number of posts: X</h3>
+    <strong><%= User.find(com.user_id).name %></strong>
+    <p><%= com.text %></p>
```

Comment:
- [ ] Please kindly ensure that the view only displays the five most recent comments using the method that was created in the Post model class

Reply:
Thanks for the great feedback Anselem! 👍🏼